content/en/docs/Writing policies/validating-admission-policies/generate-validating-admission-policies.md

```diff
@@ -14,7 +14,35 @@ Because Kyverno can be used to generate ValidatingAdmissionPolicies and their bi
 
 To generate ValidatingAdmissionPolicies, make sure to:
 1. enable `ValidatingAdmissionPolicy` [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/).
-2. enable either `admissionregistration.k8s.io/v1alpha1` or `admissionregistration.k8s.io/v1beta1` API depending on the version of Kubernetes you are running.
+2. for 1.27, enable `admissionregistration.k8s.io/v1alpha1` API, and for 1.28 enable both `admissionregistration.k8s.io/v1alpha1` and `admissionregistration.k8s.io/v1beta1` API.
```

Comment:
I'm pretty sure for 1.28 you don't have to enable the v1alpha1 version.

Reply:
In the codebase, we generate the `v1alpha1` of VAPs. That's why it needs to be enabled along with `v1beta1`.

Reply:
Ok